### DISCUSSION TITLE
ERROR: exact duplicate record found

### DISCUSSION THREAD

**Discussion by john2893** [auroraresearchlab/netbox-dns#303] [Q&A] [answered]
I get this error when I run "dnscontrol preview --expect-no-changes". I can't seem to find the duplicate records in Netbox though.
Can I check someplace to see where this duplicate record is?

2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.174.21.111.in-addr.arpa PTR host98.test174.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.175.21.111.in-addr.arpa PTR host98.test175.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.176.21.111.in-addr.arpa PTR host98.test176.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.177.21.111.in-addr.arpa PTR host98.test177.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.178.21.111.in-addr.arpa PTR host98.test178.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.179.21.111.in-addr.arpa PTR host98.test179.wfu.edu. ttl=3600
2023/10/20 13:59:20 printIR.go:113: ERROR: exact duplicate record found: 98.180.21.111.in-addr.arpa PTR host98.test180.wfu.edu. ttl=3600

**Comment by peteeckel** [comment, marked as answer]
Are you referring to this https://github.com/StackExchange/dnscontrol?

That has nothing to do with NetBox DNS, so probably there is some piece of code that uses data in NetBox DNS to provision your DNS server software - whatever that might be. 

Since you said you can't find any duplicate records in NetBox DNS but your DNS server management complains about them, the likely cause is that provisioning software, possibly because it adds records from NetBox DNS to your DNS server without checking if they are already present.

Since the records you are seeing complained about are PTR records - did you also look in the 'Managed Records' list? Not that I have an idea how the duplicates could have been generated, but I'd rather be sure that isn't the reason.

**Reply by john2893** [reply]
Thank you for pointing me to the right direction. They were in the Managed Records.
